templates/index.j2

```diff
@@ -49,6 +49,7 @@
                     <th>File</th>
                     <th>PR</th>
                     <th>comments</th>
+                    <th>labels</th>
```

Comment:
Note that above we sort by column 3, but you inserted a new column

Reply:
now sorting by the same column as before, which is now column 4